# Assignment 03-Q4 

## TeleCall uses 4 centers around the globe to process customer order forms. They audit a certain %  of the customer order forms. Any error in order form renders it defective and has to be reworked before processing.  The manager wants to check whether the defective %  varies by centre. Please analyze the data at 5% significance level and help the manager draw appropriate inferences

## Minitab File: Costomer+OrderForm.mtw

### Assume,

### H0: Defective % is same in all the centers. 

### H1: Defective % is not same, At least one center is different.

### Now in order to help the manager we will perform chi square test as our data is categorical.

In [ ]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import chi2_contingency 

In [ ]:
df=pd.read_csv('/content/drive/MyDrive/Datasets/Costomer+OrderForm.csv')

In [ ]:
df.head()

Phillippines   Indonesia       Malta       India
0   Error Free  Error Free   Defective  Error Free
1   Error Free  Error Free  Error Free   Defective
2   Error Free   Defective   Defective  Error Free
3   Error Free  Error Free  Error Free  Error Free
4   Error Free  Error Free   Defective  Error Free

In [ ]:
des=df.describe()
des

Phillippines   Indonesia       Malta       India
count           300         300         300         300
unique            2           2           2           2
top      Error Free  Error Free  Error Free  Error Free
freq            271         267         269         280

In [ ]:
df['Phillippines'].value_counts()

Error Free    271
Defective      29
Name: Phillippines, dtype: int64

In [ ]:
df['Indonesia'].value_counts()

Error Free    267
Defective      33
Name: Indonesia, dtype: int64

In [ ]:
df['Malta'].value_counts()

Error Free    269
Defective      31
Name: Malta, dtype: int64

In [ ]:
df['India'].value_counts()

Error Free    280
Defective      20
Name: India, dtype: int64

In [ ]:
freq=des.iloc[3,0:4]
freq

Phillippines    271
Indonesia       267
Malta           269
India           280
Name: freq, dtype: object

In [ ]:
defective=[29,33,31,20]

In [ ]:
p_value=stats.chi2_contingency([[freq],[defective]])
p_value

(3.8589606858203545,
 0.2771020991233144,
 3,
 array([[[271.75, 271.75, 271.75, 271.75]],
 
        [[ 28.25,  28.25,  28.25,  28.25]]]))

In [ ]:
p_value[1]

0.2771020991233144

## Conclusion:

p_value is greater than alpha 
p-value 0.2771020991233144 > 0.05,

so, we will accept null hypothesis. 

that is H0: Defective % is same in all the centers.